docs/API.md

```diff
@@ -474,9 +475,10 @@ Example response:
 
 #### `"params"`
 
-An object with the property:
+An object with the properties:
 
-`"hostname"`: A *domain name*, required. The hostname whose IP addresses are to be resolved.
+* `"hostname"`: A *domain name*, required. The hostname whose IP addresses are to be resolved.
+* `"language"`: A [Language Tag], optional, used for validation error messages translation.
 
 
```

Comment:
Below there is a section "#### error" with no information. Could that be updated with the new error report that can be returned?

Reply:
I have updated the error section of `get_data_from_parent_zone` and `get_host_by_name`.